src/super_gradients/training/utils/detection_utils.py

```diff
@@ -423,16 +425,20 @@ def draw_box_title(
         :param y2: Y coordinate of the bottom right corner of the bounding box
         :param class_id: A corresponding class id
         :param pred_conf: Class confidence score (optional)
-        :param is_target: Indicate if the bounding box is a ground-truth box or not
-
+        :param bbox_prefix: Prefix to add to the title of the bounding boxes
+        :param is_target: Not Supported Anymore. Please use  `pred_conf` and `bbox_prefix` instead.
         """
+        if is_target:
```

Comment:
Although this method is public I think there are really low chances someone is using it aside. So it should be safe to just remove the argument completely

Reply:
Removed in [efd3579](https://github.com/Deci-AI/super-gradients/pull/1686/commits/efd357918383b7a750dc690d31cbf535d2089b2a)